src/routers/secure/scrape.py

```
@@ -576,164 +645,203 @@ async def manual_update_attributes(
+                        fresh_info = service_instance.get_torrent_info(torrent_id)
+
+                        # Find matching file
+                        for file in fresh_info.files.values():
+                            if file.filename == data.filename:
+                                if file.download_url:
+                                    download_url = file.download_url
+                                    logger.debug(
+                                        f"Resolved download_url for {data.filename}: {download_url}"
+                                    )
+                                break
+                    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)